git-pr-merge

```diff
@@ -468,9 +463,23 @@ common::merge_message() {
 	# title from the PR title and the PR number, then adding a short log of
 	# what is being merged and add a diff stat of the files changed by the merge.
 
+	# Get the PR body to be the commit message. Remove any trailers so we
+	# can re-add them after the log and diff-stat along with our own
+	# Pull-request: trailer.
+	local body trailers title pr_url
+	title="$(gh pr view --json title --jq .title "${pr_ref}")"
+	body=$(gh pr view --json body --jq .body "${pr_ref}" | tr -d '\015')
```

Comment:
you deleted the TODO above re combining multiple `gh pr view` calls - do you not care about this anymore?

Reply:
yeah because I moved two of the calls to elsewhere (since the values were use in only one function). The other function gets the body which is perhaps a bit much to combine with other values. It may be possible with the `jq` `@sh` formatter, but overall it just seemed less necessary. It's not that slow anyway I think.